src/components/map/AttitudePanel.tsx

```diff
@@ -39,6 +40,23 @@ export default function AttitudePanel({
   selectedFlight,
   isPlaying,
 }: AttitudePanelProps) {
+  const [isActive, setIsActive] = useState(false);
+
+  useEffect(() => {
+    const isTimeInRange = isFlightActive(
+      progress,
+      allTimestamps,
+      operationTimestamps,
+      selectedFlight,
+    );
+
+    if (isTimeInRange) {
+      setIsActive(true);
+    } else {
+      setIsActive(false);
+    }
```

Comment:
```suggestion
    setIsActive(isTimeInRange);
```

위와 같은 형태도 고려해볼 수 있겠습니다.